PizzaAPI/PizzaAPI/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PizzaAPI.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```

Comment:
A to nie jest domyśly controller dodawany z template solucji dla Web API ? Potrzebny jest w projekcie ?